### DISCUSSION TITLE
why can't I find the difference between result got by  DescartesMotionPlannerD  and TrajOptMotionPlanner in demo  in chain_example.cpp

### DISCUSSION THREAD

**Discussion by dbdxnuliba** [tesseract-robotics/tesseract_planning#248] [Q&A]
this is the gif use DescartesMotionPlannerD ,and the seed got by     **CompositeInstruction seed = generateSeed(program, cur_state, env_);**
![descartes_chain_example](https://user-images.githubusercontent.com/13350158/190123951-15ec36b7-06f2-4761-95d0-15a0a616607d.gif)

and the following is the gif which got by using TrajOptMotionPlanner **and the seed is the result after DescartesMotionPlannerD sovle** 
![chain_example](https://user-images.githubusercontent.com/13350158/190123858-4a699dd6-8c3e-49c2-ba89-bb625955cbcb.gif)

**Comment by marip8** [comment]
I don't think there should be a difference between the output of these planners for this particular setup. In both cases we are asking a 6-DOF robot to move to fully constrained 6-DOF Cartesian poses, and there is really only one joint configuration per waypoint that is consistent with the start position. The approach and departure are collision free via joint interpolation from the simple planner step, so they don't need to be improved by Trajopt. Also there are no other collision objects in the scene that Trajopt would try to adjust the trajectories to avoid.

What difference were you expecting to see?

**Reply by dbdxnuliba** [reply]
ok ,I will try other environment to compare the results between planners
